### DISCUSSION TITLE
Is gitter.im/python-lifelines defunct?

### DISCUSSION THREAD

**Discussion by MarkPundurs** [CamDavidsonPilon/lifelines#1210] [General]
I went to https://gitter.im/python-lifelines/ today for the first time in several weeks and found a 404 error message. Is that board defunct?

**Comment by CamDavidsonPilon** [comment]
Hey, yes! I made an announcement on the gitter a few weeks ago that I want to move discussions to, well, here. A few days ago I deleted the room.

**Reply by MarkPundurs** [reply]
Thanks, @CamDavidsonPilon !
